release-tools/src/main/java/org/springframework/data/release/jira/JiraComponent.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.data.release.jira;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/*
+ * Value object to bind REST responses to.
+ * 
+ * @author Mark Paluch
+ */
+@Data
+@JsonInclude(Include.NON_NULL)
```

Comment:
`IssueTrackerConfiguration.jacksonObjectMapper()` would be the point to look at. Currently it would affect both Git an JIRA communication but I'm not sure the customization would do any harm to Git communication.